packages/react-ui-core/src/Form/Email.js

```diff
@@ -12,7 +12,6 @@ export default class Email extends PureComponent {
   static defaultProps = {
     name: 'email',
     type: 'email',
-    placeholder: 'Email Address',
```

Comment:
i see we are omitting placeholder as a prop, does this still mean placehold is available as in attribute with ...props?

Reply:
https://github.com/rentpath/react-ui/blob/af25ce3495924b396fa25f5b65bb7a4bea9a50e9/packages/react-ui-core/src/Form/Email.js#L19

Reply:
Yeah, it's still available, but it seems a little weird to have to do `placeholder: ''` just to get it to go away in ag.js

Reply:
ag.js forms don't use them.  I think it should be up to the app to decide.